gui/ForgetPassword.py

```diff
@@ -0,0 +1,46 @@
+from PyQt5.QtWidgets import QDialog
+from PyQt5.uic import loadUi
+from PyQt5 import QtCore
+from PyQt5.QtCore import Qt
+import re
+from Saqer.gui import ForgetPassSuccess
```

Comment:
I think you need to make this change because it gives "ModuleNotFoundError".
```suggestion
from gui import ForgetPassSuccess
```